phase1/gce/do

```diff
@@ -8,7 +8,7 @@ set -x
 cd "${BASH_SOURCE%/*}"
 
 generate_token() {
-  perl -e 'printf "%06x:%08x%08x\n", rand(0xffffff), rand(0xffffffff), rand(0xffffffff);'
+  perl -e 'printf "%06x.%08x%08x\n", rand(0xffffff), rand(0xffffffff), rand(0xffffffff);'
```

Comment:
Yeah, I was pretty sad that it was difficult to use here (I was the one who first [added](https://github.com/kubernetes/kubernetes/pull/35144) the generate command), but since this code is executed client-side before bringing up the cluster, there's extra friction in fetching the newest kubeadm artifacts in order to use the command locally. This is definitely something I'd like to do in the future, though.